tools/concepts/mfcc/test_mfcc_kaldi.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2022 Intel Corporation. All rights reserved.
+
+import torch
+import torchaudio
+import torchaudio.compliance.kaldi as K
+import numpy
+import sys
+
+AUDIO_SAMPLE = sys.argv[1]
+#AUDIO_SAMPLE = "test_sine.wav"
+#AUDIO_SAMPLE = "test_chirp.wav"
+MFCC_CSV = "kaldi.csv"
+
+metadata = torchaudio.info(AUDIO_SAMPLE)
+print(metadata)
+
+waveform, sample_rate = torchaudio.load(AUDIO_SAMPLE)
+
+# Use defaults for all parameters, except
+#   preemphasis_coefficient = 0
+#   remove_dc_offset False
+#   window_type "hamming"
+
+mfcc = K.mfcc(waveform, preemphasis_coefficient = 0, remove_dc_offset = False, window_type = "hamming")
+
+# With defaults
+#mfcc = K.mfcc(waveform)
+
+print(mfcc)
+
+# Convert to numpy and export, transpose for less columns (num_ceps)
+mfcc_np = mfcc.numpy()
+cols = numpy.size(mfcc_np, 1)
+rows = numpy.size(mfcc_np, 0)
+print("MFCC data size is %d x %d\n" % (cols, rows))
+numpy.savetxt(MFCC_CSV, mfcc_np, delimiter=",")
+print("Exported MFCC to %s\n" % MFCC_CSV)
```

Comment:
I just added main() into the two Python scripts. Now pylint score at 5.83/10 :)

Reply:
The only pylint issues are: 
- Module 'torchaudio' has no 'info' member
- Module 'torchaudio' has no 'load' member

but the members info/load do exist, maybe we can silent the lint error by `# pylint: disable=E1101`

Reply:
I researched this a bit and yes, `# pylint: disable=E1101` is the correct answer for the 3 functions `load`, `info` and `save`. That's because these functions are dynamically initialized depending on the OS-specific backend (sox for Linux or soundfile for Windows)

From https://github.com/pytorch/audio/blob/d62875cc6/torchaudio/backend/utils.py#L51
```python
    for func in ["save", "load", "info"]:
        setattr(torchaudio, func, getattr(module, func))
```
`setattr` is one of the usual pylint enemies.